pkg/systemd/terminal.jsx

```diff
@@ -51,6 +57,13 @@ const _ = cockpit.gettext;
             this.setState({ title: title });
         }
 
+        onThemeChanged(value) {
+            this.setState({ theme: value });
+            var cookie = "CockpitTerminalTheme=" + encodeURIComponent(value) +
+                         "; path=/; expires=Sun, 16 Jul 3567 06:23:41 GMT";
```

Comment:
Why now a cookie instead of localstorage? Because it's easier to manage by the user? (Also, what is this magic time?)

Reply:
LocalStorage gets clean on logout. https://github.com/cockpit-project/cockpit/blob/master/src/base1/cockpit.js#L2399
(Just now I looked properly on function `setItem` and it is possible to set it in localStorage that way that is survives logout). But Cockpit saves language selection in cookie and all the code is picked from there.
But now I am thinking I should probably use localStorage and save it in a way that survives logout.

Reply:
Don't get me wrong, I'm fine with a cookie. It's also more obvious for a user when searching/cleaning cookies.